DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/credit-and-debit-cards/maestro/how-does-maestro-work.md

aliases: [/payment-methods/maestro/how-does-maestro-work/]
---
## How it works

> Maestro payments are processed as a credit card payment. Therefore, credit card payments must be enabled within your MultiSafepay Control to process a Maestro payment. Submit your request to enable credit cards (Visa, Mastercard and Maestro) to your account manager or email us at <sales@multisafepay.com>

Reviewer: ```suggestion
Maestro payments are processed as credit card payments. To enable credit card payments in your MultiSafepay Control, email your account manager at <sales@multisafepay.com>
```
